arches_lingo/src/arches_lingo/pages/SchemePage.vue

```diff
@@ -34,7 +34,7 @@ const onClose = () => {
 };
 
 const onOpenEditor = (tab: string, tileId: string) => {
-    editorTab.value = tab;
+    editorForm.value = tab;
```

Comment:
`tab` language here too. Sorry to be nitpicky but this is the ticket that purges the concept of tabs from the frontend yeah?

Reply:
Yeap.  Thanks for finding those.  🏓